tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -101,8 +119,32 @@ sub run {
     repl_module_in_sourcefile();
 
     $self->{'package_name'} = 'Guest Upgrade Test';
-    $self->execute_script_run("[ -d /var/log/qa/ctcs2 ] && rm -r /var/log/qa/ctcs2 ; [ -d /tmp/prj4_guest_upgrade ] && rm -r /tmp/prj4_guest_upgrade", 30);
+    if (is_s390x) {
+        script_run "[ -d /var/log/qa/ctcs2 ] && rm -r /var/log/qa/ctcs2 ; [ -d /var/lib/libvirt/images/prj4_guest_upgrade ] && rm -r /var/lib/libvirt/images/prj4_guest_upgrade /tmp/full_guest_upgrade_test-* /tmp/kill_zypper_procs-* /tmp/update-guest-*";
+    }
+    else {
+        $self->execute_script_run("[ -d /var/log/qa/ctcs2 ] && rm -r /var/log/qa/ctcs2 ; [ -d /tmp/prj4_guest_upgrade ] && rm -r /tmp/prj4_guest_upgrade", 30);
+    }
     $self->run_test($timeout, '', 'yes', 'yes', '/var/log/qa/', $upload_log_name);
+    #upload testing logs for s390x guest upgrade test
+    if (is_s390x) {
+        #upload s390x_guest_upgrade_test.log
+        upload_asset("/tmp/s390x_guest_upgrade_test.log", 1, 1);
+        script_run "rm -rf /tmp/s390x_guest_upgrade_test.log";
+    }
```

Comment:
Shall this be removed, since it is in post_fail_hook already?

Reply:
no, s390x GU Pass still need it. there was the different part.